### PR TITLE
Rework Getting Started Page

### PR DESCRIPTION
**Pull request by 2shea**
This PR is based on feedback from new users to Talon described in https://github.com/TalonCommunity/Wiki/issues/170. In general, the improvements intend to simplify the Getting Started page and reduce friction for new users. Changes include:

* Moving content like `Speech Engines` and `Basic Usage` to their own pages. This has the additional benefit of making them easier to link to and we can add those explicitly to the nav section.
* Write instructions with defaults for the speech engine and Talon version. Still leave notes for awareness that there are alternatives, but every choice a new user has to make adds friction, and most users will want to start with the free version of Talon and the Conformer engine anyway.
* Add installation instructions to Getting Started. While some of this information is duplicated from https://talonvoice.com/docs, redirecting users to that page added friction for new users.

### CODE REVIEW

File: getting_started.md [hunk 1]
```diff
@@ -1,68 +1,36 @@
...
-The official installation and getting started instructions are available at [talonvoice.com/docs](https://talonvoice.com/docs/). We strongly recommend you follow the instructions there; this page contains additional advice only. The basic installation flow is as follows:
-
-1. Install Talon.
-1. Choose a [speech recognition engine](#configure-a-speech-recognition-engine).
-1. Install a [Talon user file set](#install-a-talon-user-file-set), to make some commands available.
-1. Try some [basic commands](#basic-usage)
-1. Optionally, join the [Talon Voice Slack](https://talonvoice.com/chat) for help, tips, and connecting with other Talon users.
-
 ## Install Talon
+1. Download and install [Talon](https://talonvoice.com) for your operating system. **Note:** For users who want earlier access to new features, priority support, and access to additional [speech engines](/speech_engines.md), install the [beta version](/beta_talon.md).
```
Comment (splondike) [inline getting_started.md:4]:
```suggestion
You can install and run Talon by following the [official installation instructions](https://talonvoice.com/docs/). The rest of this page provides some additional information to supplement the information provided there.
```

File: getting_started.md [hunk 2]
```diff
@@ -1,68 +1,36 @@
...
+1. Run the Talon app.
 
-Talon has two mostly compatible current versions: public and beta. Both versions have support for Mac, Linux, and Windows.
-
-* **Public Version (free):** Follow [Getting Started](https://talonvoice.com/docs/index.html#getting-started) instructions in the Talon documentation.
-* **Beta Version (requires Patreon support):** The beta version has earlier access to new features and higher priority support. It is required for access to some speech engines. After becoming a [beta tier Patreon](https://www.patreon.com/join/lunixbochs), join the [Talon Voice Slack](https://talonvoice.com/chat) and request access to the `#beta` channel from `@aegis`, the developer of Talon. Download links can be found in the #beta channel's pinned messages.
-
-### Notes on Linux
+**Note:** There are installation and getting started instructions available at [talonvoice.com/docs](https://talonvoice.com/docs/) which are maintained by the developer of Talon. This wiki is community maintained and aims to provide richer detail for a wider set of users, but may not always be as freshly updated with changes.
```
Comment (splondike) [inline getting_started.md:7]: I think we can simplify this by just moving it all into the previous section.
```suggestion
```

File: getting_started.md [hunk 3]
```diff
@@ -1,68 +1,36 @@
...
-
-#### Wayland
-
-You might encounter the following error:
+If you use Wayland, you might encounter the following error:
 ```
 ERROR cannot get _NET_CURRENT_DESKTOP
 ```
-In which case it is necessary to switch to Xorg. (Your distro might support this through a cog wheel on the login screen.)
+In which case it is necessary to switch to Xorg. (Your disdribution might support this through a cog wheel on the login screen.)
```
Comment (splondike) [inline getting_started.md:16]:
```suggestion
In which case it is necessary to switch to Xorg. (Your distribution might support this through a cog wheel on the login screen.)
```

File: getting_started.md [hunk 4]
```diff
@@ -77,147 +45,14 @@ cd "%APPDATA%\Talon\user"
...
+Test your Talon setup with a few [basic commands](/basic_usage.md).
 
+## Join the Talon Slack
+If at any point you need help with your Talon setup, join the [Talon Voice Slack](https://talonvoice.com/chat) and ask in the #help channel.
 
 ## Next Steps
-
 Once the basics somewhat work for you, you'll likely want to improve your experience using Talon:
-
 * [Improve Recognition Accuracy](/improving_recognition_accuracy): Better accuracy never hurts. Many people have to tweak something.
```
Comment (splondike) [inline getting_started.md:56]:
```suggestion
* For users who want earlier access to new features, priority support, and access to additional [speech engines](/speech_engines.md), install the [beta version](/beta_talon.md).
* [Improve Recognition Accuracy](/improving_recognition_accuracy): Better accuracy never hurts. Many people have to tweak something.
```

**Review by splondike — APPROVED** [review]
Thanks, this is an improvement I think! Just added a couple of small reordering suggestions for your consideration.

File: getting_started.md (hunk 4 above)
Comment (2shea) [reply getting_started.md:56]: ah, good idea adding this mention of the beta here.

**Pull request merged**